client/packages/system/src/Vaccination/api/useVaccination.ts

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import { useQuery } from '@openmsupply-client/common';
+
+import { Clinician } from '../../Clinician';
+import { useVaccinationsGraphQL } from './useVaccinationsGraphQL';
+import { VACCINATION } from './keys';
+
+export interface VaccinationDraft {
```

Comment:
Okay, good, you've started a new API group. I'll use this for the Card queries too.